### PR TITLE
Release - 1.2.6

### PR DESCRIPTION
**Pull request by nivida**
## Description

This release does add the Görli testnet to the ENS config and does update the Registry address.

This is a security patch release and should get published today.

[Release Guideline](https://github.com/ethereum/web3.js/blob/1.x/CONTRIBUTIONS.md#releases)

### Added

- Görli testnet ENS registry added to the known registries (#3338)

### Changed

- ENS registry addresses updated (#3353, https://medium.com/the-ethereum-name-service/ens-registry-migration-bug-fix-new-features-64379193a5a)

### Compare View

[v1.2.5 -> v1.2.6](https://github.com/ethereum/web3.js/compare/v1.2.5...release/1.2.6)

## Type of change

- [x] Release

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran ```npm run test``` with success and extended the tests if necessary.
- [x] I ran ```npm run build``` and tested the resulting file from ```dist``` folder in a browser.
- [x] I have tested the minified file locally

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -125,4 +125,14 @@ Released with 1.0.0-beta.37 code base.
...
 
+## [1.2.6]
+
+### Added
+
+- ENS Görli testnet registry added 
+
+### Changed
+
+- ENS registry addresses updated
```
Comment (holgerd77) [inline CHANGELOG.md]: The release notes should give a bit more context on what is happening here and why this is changed, eventually also linking to a reliable external source.

If the changes are done here in the in the release PR itself, there should be a self-reference to this PR #3353 at the end of the change items.

**Review by cgewecke — APPROVED** [review]
LGTM!

**Review by michaelsbradleyjr — APPROVED** [review]

**Review by Arachnid — APPROVED** [review]
Please do whatever is necessary to update web3 - users who do not update will start seeing stale results on Monday, and we still need to encourage developers to use the new library once it is released!

**Comment by nivida** [comment]
@holgerd77 @michaelsbradleyjr @cgewecke @Arachnid version 1.2.6 of web3.js got successfully released.

**Comment by nivida** [comment]
Closed because PR https://github.com/ethereum/web3.js/pull/3355 will apply those required changes on 1.x without reverting of improvements we already have done.

**Pull request closed without merging**